templates/firefox-ios.xliff

```diff
@@ -1575,6 +1571,35 @@
         <source>URL (Replace Query with %s)</source>
         <note>The placeholder for URL Field when saving a custom search engine</note>
       </trans-unit>
+      <trans-unit id="Settings.AdvanceAccount.ErrorAlertMessage">
+        <source>There was an error while attempting to parse the url. Please make sure that it is a valid Firefox Account root url.</source>
+        <note>Messaged displayed when sync service has an error setting a custom sync url.</note>
+      </trans-unit>
+      <trans-unit id="Settings.AdvanceAccount.ErrorAlertOk">
+        <source>OK</source>
+        <note>Ok button on custom sync service error alert.</note>
+      </trans-unit>
+      <trans-unit id="Settings.AdvanceAccount.ErrorAlertTitle">
+        <source>Error</source>
+        <note>Error alert message title.</note>
+      </trans-unit>
+      <trans-unit id="Settings.AdvanceAccount.SectionName">
+        <source>Account Settings</source>
+        <note>Label used as an item in Settings. When touched it will open a dialog to setup advance Firefox account settings.
+   Title displayed in header of the setting panel.</note>
+      </trans-unit>
+      <trans-unit id="Settings.AdvanceAccount.UpdatedAlertMessage">
+        <source>Firefox account service updated. To begin using custom server, please log out and re-login.</source>
```

Comment:
Should this be "Firefox Accounts service updated"? For sure it's inconsistent with how it's written elsewhere.
Firefox Account -> one Firefox Account
Firefox Accounts -> name of the service

Also: login is a name, log in a verb. Was this copy reviewed? Because "please log out and log in again" (or try again) seems a lot easier.

Reply:
This was not reviewed.